### PR TITLE
Remove fancy part around 'make core' in the Makefile

### PR DESCRIPTION
**Pull request by aryx**
I don't get anymore the progress display of dune
and this has the bad effect also of blocking my editors efuns
when running 'make core' from it

test plan:
make
=> see dune build progress

### CODE REVIEW

**Review by ajbt200128 — CHANGES_REQUESTED** [review]
I'm not against this change, I just think we should at least keep a comment or something with the link help text. I'm just worried about people running into linker errors and not knowing that it can be expected, nor how to fix them

**Comment by aryx** [comment]
If you look at the output in case of linking error in CI, there is now some mention of src/main/flags.sh and linking errors
that I think will be enough for people to fix the issue.

**Review by ajbt200128 — APPROVED** [review]
yea this seems fine, but we probably want @kopecs to take a look at this to make sure

File: src/main/flags.sh [hunk 1]
```diff
@@ -154,7 +154,7 @@ else
             "-lbase_stubs"
             "-lbigstringaf_stubs"
             "-lca_certs_stubs"
-            "-lcamlstr"
+            #"-lcamlstr"
```
Comment (ajbt200128) [inline src/main/flags.sh]: why is this commented out now?

Comment (aryx) [reply src/main/flags.sh]: on purpose, to generate errors in CI one can inspect, to decide whether or not the current error output is enough to be directed towards src/main/flags.sh

I'll remove the '#' once this PR is approved.

**Comment by aryx** [comment]
This is the current error in CI:
image

**Comment by aryx** [comment]
and if you scroll up you even get 
image

File: src/main/flags.sh (hunk 1 above)
Comment (ajbt200128) [reply src/main/flags.sh]: ahh ok

**Pull request merged**